From 015537b88cfc7b918bff43c02ce2973a45d59bcf Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Mon, 28 Mar 2016 14:56:29 +0300
Subject: [PATCH 283/538] localversion: fix modules_install

localversion must not include a space, if it does the
modules_install will fail since it tries to create a link
to a directory that it partially created.

Change-Id: I29f62b7857bb54ea4ed4b4895c3fde3d2b2bd69d
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28598
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-by: Lior Amsalem <alior@marvell.com>
---
 localversion | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion b/localversion
index f314d69..872e3c7 100644
--- a/localversion
+++ b/localversion
@@ -1 +1 @@
--16.03.1 ED
+-devel-16.03.1
-- 
1.9.1

